README.md

```diff
@@ -1,4 +1,4 @@
-# darts: Easy manipulation and forecasting of time series
+# Darts: Time Series Made Easy in Python
```

Comment:
What about removing "Darts:" and keep only "Time Series Made Easy in Python" from the title since we know its darts and it is just under ?
![image](https://user-images.githubusercontent.com/66320848/85712444-9fed0500-b6e8-11ea-8ee2-bea5d07d4ada.png)

Reply:
@hrzn

Reply:
Sure, good idea.